src/common/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp

```diff
@@ -405,12 +405,14 @@ bool ov::pass::GroupedSliceToVSplitOptimization::run_on_model(const std::shared_
         auto variadic_split = std::make_shared<op::v1::VariadicSplit>(output, axis_const, split_lengths_const);
 
         auto i = 0;
+        NodeVector ops_to_replace;
         for (auto& slice_with_attrs : attributes) {
-            graph_rewritten |=
-                ov::replace_output_update_name(slice_with_attrs.slice->output(0), variadic_split->output(i));
-            ov::copy_runtime_info(slice_with_attrs.slice, variadic_split);
+            graph_rewritten = true;
+            slice_with_attrs.slice->output(0).replace(variadic_split->output(i));
```

Comment:
".replace" method doesn't preserve some legacy names (friendly names, legacy tensor names)  unlike replace_output_update_name method.
@ilyachur do we still need to preserve these legacy names?

Reply:
@itikhono Method `replace` is used in lot of transformations meanwhile.

Reply:
The same transformation several linses above https://github.com/openvinotoolkit/openvino/blob/5f9c913d6444f27c311bdf73ba9885b334eedc71/src/common/transformations/src/transformations/common_optimizations/optimize_strided_slice.cpp#L256

Reply:
I think you should still use replace_output_update_name. Otherwise if Slice output is a model output, you can encounter issues like: "Cannot find blob with name:"

> The same transformation several lines above

That can be an issue as well.